test/utils/Counters.test.js

```diff
@@ -9,23 +9,26 @@ describe('Counters', function () {
   beforeEach(async function () {
     this.counter = await CountersImpl.new();
   });
-
-  it('starts at zero', async function () {
-    expect(await this.counter.current()).to.be.bignumber.equal('0');
+  context('once deployed', function () {
```

Comment:
We should only add a `once deployed` context for contracts where we have some tests "before" deployment, for example if we have to test that deployment fails because of invalid constructor arguments.

I think all of the `once deployed` contexts you added in this PR should be removed. It's okay if an `it` is not inside a `context`.